CONTRIBUTING.md

```diff
@@ -40,28 +40,46 @@ If you want to debug the WAR file without using Maven plugins,
 You can run the executable with [Remote Debug Flags](https://stackoverflow.com/questions/975271/remote-debugging-a-java-application)
 and then attach IDE Debugger to it.
 
-To launch a development instance, after the above command, run:
+### Launching a development instance
+
+To launch a development instance, after [building the WAR file](#building-the-war-file), run:
 
 ```sh
-mvn -pl war jetty:run
+MAVEN_OPTS='--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED' mvn -pl war jetty:run
```

Comment:
Compare https://github.com/jenkinsci/maven-hpi-plugin/pull/507. If I follow correctly, the root problem is that we are trying to run the controller in the same JVM as Maven, which also causes other issues (for example `mvnd` does not work). Perhaps it would be better to fork a JVM, taking into account developer workflows with debuggers and resource reloading.

Reply:
> Perhaps it would be better

That would certainly be better, if someone wishes to work on that

Reply:
Something else I found today, FTR: `hpi:run` on a plugin using a `commons-compress-api` dep (1.26.1) does not necessarily work, because `commons-compress:1.21` is loaded from the mojo instead.